.docker/php-dev.ini

```diff
@@ -0,0 +1 @@
+opcache.validate_timestamps = 1
```

Comment:
Could use environment variables instead. Don't think there's a way to set a default though.